docs/docs/guide/installing-filters.md

```diff
@@ -111,4 +111,15 @@ Alternatively, you can use the `--force-resolver-update` flag to force the resol
 
 ```
 regolith install name_ninja --force-resolver-update
+```
+
+### Updating filter cache
+
+Regolith caches the filter repository when you install online filters. By default, the repository cache is updated every 5 minutes. 
```

Comment:
I think this sentence is a little bit misleading. Maybe something like this would be better?:

> Regolith caches the filter repository when you install online filters. To avoid unnecessary frequent updates, Regolith skips them for installations that occur less than 5 minutes after the last update.